### PR TITLE
v0.26 release leads 

### PR DESCRIPTION
**Pull request by lionelvillard**
# Changes

- Set @ZhiminXiang and @evankanderson as v0.26 release leads
-
-

 

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Review by evankanderson — COMMENTED** [review]
Can you (manually) run `scripts/update-gen.sh` (I think) to update OWNERS_ALIASES as well?

**Comment by lionelvillard** [comment]
done! (ran `hack/update-codegen.sh`)

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -114,8 +113,8 @@ aliases:
   - vaikas
   knative-release-leads:
   - RichieEscarez
-  - lionelvillard
-  - yanweiguo
+  - ZhiminXiang
```
Comment (vaikas) [inline OWNERS_ALIASES:116]: Just curious on why / if we still need @RichieEscarez here as a release lead. My understanding was that it was earlier required for Hugo related tasks, but release leads have been able to do this for a few releases now. See below also.

File: peribolos/knative.yaml [hunk 2]
```diff
@@ -877,8 +877,8 @@ orgs:
           Knative Release Leads:
             description: Active member of the Knative Release Leads rotation.
             maintainers:
-            - yanweiguo
-            - lionelvillard
+            - ZhiminXiang
+            - evankanderson
```
Comment (vaikas) [inline peribolos/knative.yaml:881]: I think the next issue has now been done, so I believe we can remove that entry now and solely rely on the Knative Release Leads and no special docs permissions are required.

Comment (lionelvillard) [reply peribolos/knative.yaml:881]: not sure I understand. Is there anything to do other than removing Richie (which I did)?

Comment (vaikas) [reply peribolos/knative.yaml:881]: Nope, that was all I thought we could now safely do. I just jotted it first down in the first file, then pointed to place where (which you did in this commit) the change needed to be made.
Thanks and sorry for the confusing verbiage :)

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -69,7 +69,7 @@ aliases:
   - pierDipi
   - vaikas
   knative-admin:
-  - RichieEscarez
+  - ZhiminXiang
```
Comment (vaikas) [inline OWNERS_ALIASES:72]: I don't actually think we should be making any changes as part of this PR (for the release) to the knative-admin. Only the knative-release-leads should change.
Revisiting the knative-admin roles (I think) should be done as a separate PR.

Comment (lionelvillard) [reply OWNERS_ALIASES:72]: hmm this is automatically generated.... maybe there is a bug in the script?

Comment (vaikas) [reply OWNERS_ALIASES:72]: nah, you're right as usual. Been a long day. Sorry for the noise :)
